src/App.js

```diff
@@ -80,7 +82,14 @@ function App() {
                 key={drink.idDrink}
                 path={`${drink.idDrink}`}
                 element={
-                  <DrinkPage drink={drink} toggleBookmark={toggleBookmark} />
+                  <DrinkPage
+                    drink={drink}
+                    toggleBookmark={toggleBookmark}
+                    saveRatingToDrink={saveRatingToDrink}
+                    handleSubmitComment={handleSubmitComment}
+                    commentValue={commentValue}
+                    setCommentValue={setCommentValue}
+                  />
```

Comment:
Wouldn't it make more sense to put `DrinkPage` in the folder `pages` and not in the folder `components`? 🤔

Reply:
Yes, I need to refactor the whole DrinkPage component to be able to reuse it for the random drink page too. This will be part of single refactoring user story.